eng/Version.Details.xml

```diff
@@ -170,14 +170,9 @@
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>eb809c8854de4d1bdf227a55550af01a1bb38e4b</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build" Version="0.1.0-alpha.1.21616.4" CoherentParentDependency="Microsoft.NET.Sdk">
-      <Uri>https://github.com/dotnet/source-build</Uri>
-      <Sha>90bdf447e1b97605f109b34243ab8c9f215308e9</Sha>
-      <SourceBuild RepoName="source-build" ManagedOnly="true" />
-    </Dependency>
-    <Dependency Name="Microsoft.Deployment.DotNet.Releases" Version="1.0.247101">
+    <Dependency Name="Microsoft.Deployment.DotNet.Releases" Version="1.0.0-preview4.1.22206.3">
```

Comment:
Updating deployment-tools is necessary because the old version was referencing the source-build repo.